common/app/views/support/JavaScriptPage.scala

```diff
@@ -30,6 +30,7 @@ case class JavaScriptPage(metaData: MetaData)(implicit request: RequestHeader) {
       ("edition", JsString(edition.id)),
       ("ajaxUrl", JsString(Configuration.ajax.url)),
       ("isDev", JsBoolean(Play.isDev)),
+      ("isProd", JsBoolean(Play.isProd)),
```

Comment:
Was actually thinking `("isProd", JsBoolean(Configuration.environment.isProd))`
but this is better if it works. :+1: